cvmfs/hash.h

```diff
@@ -393,7 +393,7 @@ unsigned GetContextSize(const Algorithms algorithm);
 
 /**
  * Holds an OpenSSL context, only required for hash operations.  Allows to
- * deferr the storage allocation for the context to alloca.
+ * deferr the storage allocation for the context to allocate.
```

Comment:
alloca is correct here.  The `alloca()` routine allocates space on the stack, instead of the heap.